components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/store/SyncState.kt

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.fxa.store
+
+import mozilla.components.concept.sync.Avatar
+import mozilla.components.concept.sync.ConstellationState
+import mozilla.components.concept.sync.OAuthAccount
+import mozilla.components.concept.sync.Profile
+import mozilla.components.lib.state.State
+
+/**
+ * Global state of sync.
```

Comment:
```suggestion
 * Global state of Firefox Sync.
```

nit: `Sync` or `Firefox Sync`.